src/localization/en.js

```diff
@@ -188,7 +200,65 @@ export default {
       },
     },
 
-    currentExpenses: {},
+    currentExpenses: {
+      header:                          'Current Household Expenses',
+      unreimbursedNonMedicalChildCare: {
+        sectionHeading:    'Reasonable Unreimbursed Non-Medical Child(ren) Care',
+        subheading:        'A "child" is a person 12 or younger. Don\'t include amounts that are paid for by other benefit programs.',
+        columnExpenseType: 'Expense',
+        childDirectCare:   {
+          label: 'Direct care costs',
+          hint:  'How much do you pay for child care out of pocket?',
+        },
+        childBeforeAndAfterSchoolCare: {
+          label: 'Before- and after-school care',
+          hint:  'How much do you pay for child care for times before or after school?',
+        },
+        childTransportation: {
+          label: 'Transportation costs',
+          hint:  'How much do you pay for transportation?',
+        },
+        childOtherCare: {
+          label: 'Other care',
+          hint:  'How much do you pay for other child care?',
+        },
+        earnedBecauseOfChildCare: 'Does child care allow you to make additional income?',
+        ifYes:                    'Income made possible by childcare expenses',
+      },
+      childSupport: {
+        sectionHeading:      'Child Support',
+        columnExpenseType:   'Expense',
+        childSupportPaidOut: 'Legally obligated child support',
```

Comment:
We should indicate that "Legally obligated" is bold somehow, either using interpolation of a `<strong>` tag (like in the about page), or maybe using Markdown as I suggested in slack.

Reply:
Indeed - that already is in its own node. I'd suggest interpolation for now if we want to get this done more quickly.